### ISSUE TITLE
Unit of chamfer loss

### ISSUE THREAD

**Issue by rakeshshrestha31** [noahcao/Pixel2Mesh#15]
Thanks for sharing the code. 

I have a question about the unit of chamfer loss. The chamfer loss computed in the code is in meters, right?. But the one reported in the README seems to be in a different unit (around 0.4, which looks too large to be in meters). Any reason for the discrepancy?

Also, during training should we scale the chamfer loss by 3000 as done in the original tensorflow implementation for the training process as well? Reference:
https://github.com/nywang16/Pixel2Mesh/blob/40d7c6fb2cc709db32b866f6a72003f90bb3b8c2/p2m/losses.py (line 58)

**Comment by ultmaster** [comment]
First, the code always use meters. Reported in README are, I don't remember exactly whether we report the number in millimeters or centimeters, but it is to align with the scale in papers (both Pixel2Mesh and some follow-up works).

Second, you might've noticed that the original tf implementation got a really messy loss implementation, and it's definitely not said in paper that loss should be scaled by 3000. Although we've implemented a similar thing in our config to scale the loss:

https://github.com/noahcao/Pixel2Mesh/blob/1d5011928fc209a1a67a59fbe1be62980db1884f/options.py#L65

we didn't use it. Instead we tried to scale the learning rate. The performances are on par.

**Comment by rakeshshrestha31** [comment]
Thank you for the prompt response.  Just a simple follow-up, the learning rate you used for training the code is 0.0001 (1e-4) for the resnet backbone while the original tensorflow implementation uses 1e-5. A factor of 10 was enough to compensate the loss scaling of 3000 in the original code?

**Comment by ultmaster** [comment]
Actually no.

You are speaking of another weird thing. I myself got confused in this scaling problem for a week or two and didn't get a clue eventually. I'm guessing it's due to some subtle differences between the initialization and adam optimizer implementation between tensorflow and pytorch. But I don't have that much time to dig into it afterwards. Also I noticed that this task is very sensitive to learning rate. This 1e-4 is a learning rate I've tuned manually after a lot of trial-and-error...

**Comment by rakeshshrestha31** [comment]
Thank you for the explanation. I am closing this issue now.

**Comment by zshyang** [comment]
Hi,
Do you know how to download the dataset and unzip it from the link below?
https://drive.google.com/open?id=131dH36qXCabym1JjSmEpSQZg4dmZVQid

**Comment by zshyang** [comment]
Hi,
Do you know how to use their provided YAML files and pretrained weights downloaded from here to do the evaluation?
To my understanding, I should use the downloaded meta folder. Then use the provide pretrained weights.
The command I use is 
```
python entrypoint_eval.py --name xxx --options experiments/default/tensorflow.yml --checkpoint datasets/data/pretrained/vgg16-p2m.pth 
```
But it seems that it only generated some meaningless meshes. And the score is really low.
I am not sure what I did wrong.
![Screenshot from 2020-09-22 02-08-41](https://user-images.githubusercontent.com/57812785/93863819-de0eac00-fc78-11ea-91b5-0235f5b4487b.png)
